src/test/java/io/confluent/idesidecar/restapi/integration/LocalITs.java

```diff
@@ -0,0 +1,267 @@
+package io.confluent.idesidecar.restapi.integration;
+
+import io.confluent.idesidecar.restapi.kafkarest.RecordsV3ApiErrorsSuite;
+import io.confluent.idesidecar.restapi.kafkarest.RecordsV3ApiSuite;
+import io.confluent.idesidecar.restapi.kafkarest.api.ClusterV3Suite;
+import io.confluent.idesidecar.restapi.kafkarest.api.PartitionV3Suite;
+import io.confluent.idesidecar.restapi.kafkarest.api.TopicConfigV3Suite;
+import io.confluent.idesidecar.restapi.kafkarest.api.TopicV3Suite;
+import io.confluent.idesidecar.restapi.messageviewer.SimpleConsumerSuite;
+import io.confluent.idesidecar.restapi.resources.KafkaConsumeSuite;
+import io.confluent.idesidecar.restapi.testutil.NoAccessFilterProfile;
+import io.confluent.idesidecar.restapi.util.LocalTestEnvironment;
+import io.confluent.idesidecar.restapi.util.TestEnvironment;
+import io.quarkus.test.junit.QuarkusIntegrationTest;
+import io.quarkus.test.junit.TestProfile;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Tag;
+
+public class LocalITs {
+
+  private static final LocalTestEnvironment TEST_ENVIRONMENT = new LocalTestEnvironment();
+
+  @BeforeAll
+  static void beforeAll() {
+    // Start the test environment and use it for all tests
+    TEST_ENVIRONMENT.start();
+  }
+
+  @AfterAll
+  static void afterAll() {
+    // Shutdown the test environment after all tests have run
+    TEST_ENVIRONMENT.shutdown();
+  }
+
+  @Nested
+  class LocalConnectionTests {
```

Comment:
A `@Nested` test class exists for each kind of connection to be tested. This is for local connections. This could just implement all of the IT suite interfaces, but instead we define them on multiple `@Nested` test classes in this test class.